### PR TITLE
Try to fix package generation for MAcOs Arm64

### PR DESCRIPTION
**Pull request by pimpin**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -237,7 +237,7 @@ jobs:
     needs:
       - source
       - build_macos_arm64
-    runs-on: macos-14
+    runs-on: macos-14-arm64
```
Comment (avsej) [inline .github/workflows/tests.yml:240]: @pimpin does it work? according to official page there is no such runner:

https://github.com/actions/runner-images/blob/main/README.md#available-images

**Comment by avsej** [comment]
Oh, I see https://rubygems.org/gems/couchbase/versions does not have `arm64-darwin` for 3.5.2.

I will make sure that 3.5.3 will have it published. I'm working on the release right now

File: .github/workflows/tests.yml (hunk 1 above)
Comment (pimpin) [reply .github/workflows/tests.yml:240]: Indeed it's not listed in the README, but it is listed in MacOs images : https://github.com/actions/runner-images/blob/main/images/macos/macos-14-arm64-Readme.md

**Comment by avsej** [comment]
@pimpin could you try to install the latest version (3.5.3), I've just updated rubygems.org. It should automatically pick one precompiled

**Comment by avsej** [comment]
closing for now. Please reopen if you still have issues.

**Pull request closed without merging**